scripts/check-docs.sh

```
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+mkdocs build --strict
+RUSTC_WRAPPER= cargo check -p awa --example quickstart
```

Shellcheck (0.11.0)

[warning] Remove space after = if trying to assign a value (for empty string, use var='' ... ).

(SC1007)